proposals/0026-hlsl-long-vector-type.md

```diff
@@ -118,6 +118,12 @@ myBuffer.Store< vector<T, N> >(StartoffsetInBytes + 100, val);
 
 ```
 
+Long Vectors initialized from or dumped to a Raw Buffer must comply with the
```

Comment:
Minor nit, can we just say `loaded from or stored to` instead of `initialized from or dumped to`?